src/fluid.cpp

```diff
@@ -74,6 +74,56 @@ bool Fluid::IsGivenKeyMaster(CDynamicAddress inputKey) {
     return false;
 }
 
+/** Checks fluid transactoin operation script amount for invalid values. */
+bool Fluid::CheckFluidOperationScript(const CScript& fluidScriptPubKey, std::string& errorMessage) {
+    std::string strFluidOpScript = ScriptToAsmStr(fluidScriptPubKey);
+    std::string verificationWithoutOpCode = GetRidOfScriptStatement(strFluidOpScript);
+    std::string strOperationCode = GetRidOfScriptStatement(strFluidOpScript, 0);
+    if (!ExtractCheckTimestamp(strFluidOpScript, GetTime())) {
+        errorMessage = "CheckFluidOperationScript fluid timestamp is too old.";
```

Comment:
Range can be in past and in future, should be "out of range" error

Reply:
Yeah, you are correct.  I think I should change GetTime() to the transaction time and causing this error:
`ERROR: CheckBlock(): CheckFluidOperationScript fluid timestamp is too old....`

Reply:
I changed it to a timestamp and it fixed the issue.  thanks again.